ads/google/imaVideo.js

```diff
@@ -600,6 +602,7 @@ export function onContentPauseRequested(global) {
 export function onContentResumeRequested() {
   adsActive = false;
   videoPlayer.addEventListener(interactEvent, showControls);
+  window.parent./*OK*/postMessage({event: VideoEvents.AD_END}, '*');
```

Comment:
You'll want to add this to onAdError as well. That gets called if something goes wrong while playing an ad, and in that case we don't trigger contentResume.